src/request.ts

```diff
@@ -887,9 +867,8 @@ class DatastoreRequest {
     return stream;
   }
 
-  private getSharedQueryOptions(
-    query: Query,
-    options: RunQueryStreamOptions = {}
+  private getSharedOptionsOnly(
```

Comment:
what is the difference between this method and `getSharedQueryOptions`? The naming is confusing here.

Reply:
`getSharedQueryOptions` also accepts a query and calls `getSharedOptionsOnly`. `getSharedOptionsOnly` is also called for fetches in `createReadStream` where options need to be built without considering a query.

Reply:
Thanks for the explanation! Maybe `getSharedQueryOptions` and `getSharedRequestOptions`, or `getQueryOptions` and `getRequestOptions` would be a little more readable, wdyt?

Reply:
`getQueryOptions` and `getRequestOptions` would work really well. The good thing about these two functions is that they are both private so we can change the names later too if need be.